samples/highcharts/demo/area-step/demo.js

```diff
@@ -0,0 +1,521 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'area'
+    },
+    title: {
+        text: 'United States Fed Funds Interest Rate',
+        align: 'left'
+    },
+    xAxis: {
+        type: 'datetime'
+    },
+    yAxis: {
+        title: {
+            text: undefined
+        },
+        labels: {
+            format: '{value}%'
+        }
+    },
+    tooltip: {
+        valueDecimals: 2,
+        valueSuffix: '%'
+    },
+    series: [{
+        name: 'Interest Rate',
+        showInLegend: false,
+        step: {
+            type: 'left',
+            risers: false
+        },
+        fillOpacity: 0.25,
+        linecap: 'butt',
+        lineWidth: 3,
+        data: [
+            [
+                Date.UTC(2014, 9, 29),
```

Comment:
It will all come together when we run [remove-utc](https://github.com/highcharts/highcharts/blob/v12-staging/tools/remove-date-utc.js) before release.